__tests__/db.test.js

```diff
@@ -30,50 +31,100 @@ describe("/api/categories", () => {
   });
 });
 
-describe('/api', () => {
-    test("GET - status: 200 - responds with endpoints.json content", () => {
-        return request(app)
-        .get("/api")
-        .expect(200)
-        .then((apiResult) => {
-            return fs.readFile(`${__dirname}/../endpoints.json`, "utf-8").then((readFileResult) => {
-                expect(apiResult.body.endpoints).toEqual(JSON.parse(readFileResult));
-            })
-        })
-    });
+describe("/api", () => {
+  test("GET - status: 200 - responds with endpoints.json content", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((apiResult) => {
+        return fs
+          .readFile(`${__dirname}/../endpoints.json`, "utf-8")
+          .then((readFileResult) => {
+            expect(apiResult.body.endpoints).toEqual(
+              JSON.parse(readFileResult)
+            );
+          });
+      });
+  });
 });
 
-describe('/api/reviews/:review_id', () => {
-    test('GET - status: 200 - responds with correct review object', () => {
-        return request(app)
-        .get("/api/review/3")
-        .expect(200)
-        .then((result) => {
-            expect(result.body.review).toEqual({ review_id: 3, title: "Ultimate Werewolf", category: "social deduction", designer: "Akihisa Okui", owner: "bainesface", review_body: "We couldn't find the werewolf!", review_img_url: "https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?w=700&h=700", created_at: "2021-01-18T10:01:41.251Z", votes: 5});
-        })
-    });
-    test('GET - status: 400 - gives correct error when given an invalid review_id number', () => {
-        return request(app)
-        .get("/api/review/666")
-        .expect(404)
-        .then((result) => {
-            expect(result.body.msg).toEqual("Invalid review_id")
+describe("/api/reviews/:review_id", () => {
+  test("GET - status: 200 - responds with correct review object", () => {
+    return request(app)
+      .get("/api/review/3")
+      .expect(200)
+      .then((result) => {
+        expect(result.body.review).toEqual({
+          review_id: 3,
```

Comment:
I would suggest this only needs to assert the id keys values, and just the presence of the other keys but fine with this.